anaconda_linter/lint/__init__.py

```diff
@@ -617,17 +624,35 @@ def get_report(
         :param verbose: (Optional) Enables additional reporting.
         :returns: String, containing information about all the linting messages, as a report.
         """
-        messages = sorted(messages, key=lambda d: (d.fname, d.end_line))
-        if verbose:
-            return "\n".join(
-                f"{msg.severity.name}: {msg.fname}:{msg.end_line}: {msg.check}: {msg.title}\
-                    \n\t{msg.body}"
-                for msg in messages
-            )
-        else:
-            return "\n".join(
-                f"{msg.severity.name}: {msg.fname}:{msg.end_line}: {msg.check}: {msg.title}" for msg in messages
-            )
+        severity_data: dict[Severity, list[LintMessage]] = {sev: [] for sev in Severity}
```

Comment:
I'm sure you are aware, but let's move towards 
```suggestion
        severity_data: dict[Severity, list[LintMessage]] = {}
```
And perform the necessary check changes below.